src/nvim/misc1.c

```diff
@@ -1749,11 +1748,7 @@ del_lines (
 
 int gchar_pos(pos_T *pos)
 {
-  char_u      *ptr = ml_get_pos(pos);
-
-  if (has_mbyte)
-    return (*mb_ptr2char)(ptr);
-  return (int)*ptr;
+  return utf_ptr2char(ml_get_pos(pos));
```

Comment:
Note for next time: it's often good to have a local (`ptr`) anyway, for debugging.